pipenv/vendor/tomlkit/items.py

```diff
@@ -789,6 +789,18 @@ def append(self, key, _item):  # type: (Union[Key, str], Any) -> Table
         return self
 
     def remove(self, key):  # type: (Union[Key, str]) -> Table
```

Comment:
I don't know if I am doing anything wrong but the `parsed_pipfile['packages']` is a `tomlkit.container.Container` object and this method is not called when `del parsed_pipfile['packages']['mypackage']`.